### ISSUE TITLE
Is it a bug for "html-classes" argument?

### ISSUE THREAD

**Issue by fancofang** [trentm/python-markdown2#391]
My instance is:
```
target.body_html = markdown(value, extras=["tables", "fenced-code-blocks", "toc", "cuddled-lists", "pyshell",
                                                   "break-on-newline", "strike","html-classes"])
```
If I convert a code with `class` arg in tag, it works.

But when without `class` arg in tag, that I would get this error below:
```
Traceback (most recent call last):
...
    if tag in html_classes_from_tag:
TypeError: argument of type 'NoneType' is not iterable
```

I think the problem might be in below:
```
    def _html_class_str_from_tag(self, tag):
       ...
        try:
            html_classes_from_tag = self.extras["html-classes"]
        except TypeError:
            return ""
        else:
            if tag in html_classes_from_tag:
                return ' class="%s"' % html_classes_from_tag[tag]
        return ""
```
If `html_classes_from_tag ` is NoneType, that means the condition `if tag in html_classes_from_tag` will raise error.

**Comment by nicholasserra** [comment]
Yup sounds like a bug. If you could post some exact `value`s that would reproduce this, that would be great. Thanks!

**Comment by fancofang** [comment]
I reproduce below
```
>>> from markdown2 import Markdown
>>> md = Markdown(extras=["fenced-code-blocks","html-classes"])
>>> code1 = '''
... ```python
... print("hello world")
... ```
... '''
>>> code2 = '''
... ```
... print("hello world")
... ```
... '''
>>> md.convert("* boo!")
'<ul>\n<li>boo!</li>\n</ul>\n'
>>> md.convert(code1)
'<div class="codehilite"><pre><span></span><code><span class="nb">print</span><span class="p">(</span><span class="s2">&quot;hello world&quot;</span><span class="p">)</span>\n</code></pre></div>\n'
>>> md.convert(code2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\Anaconda3\lib\site-packages\markdown2.py", line 352, in convert
    text = self._do_fenced_code_blocks(text)
  File "D:\Anaconda3\lib\site-packages\markdown2.py", line 1852, in _do_fenced_code_blocks
    return self._fenced_code_block_re.sub(self._fenced_code_block_sub, text)
  File "D:\Anaconda3\lib\site-packages\markdown2.py", line 1848, in _fenced_code_block_sub
    return self._code_block_sub(match, is_fenced_code_block=True)
  File "D:\Anaconda3\lib\site-packages\markdown2.py", line 1797, in _code_block_sub
    pre_class_str = self._html_class_str_from_tag("pre")
  File "D:\Anaconda3\lib\site-packages\markdown2.py", line 1818, in _html_class_str_from_tag
    if tag in html_classes_from_tag:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by nicholasserra** [comment]
Fixed in #439
